### PR TITLE
Updated uploads to take in PDF, PNG and JPG with helper text

### PR DESCRIPTION
**Pull request by ReinoRabbit**
**Updated the written approval of offer to accept more document types:**

- Added helper text like on the design:

image

- Uploaded PDF:

image

- Uploaded PNG:

image

- Uploaded JPG:

image

- When uploading the wrong file type:

image

### CODE REVIEW

**Review by LlewellynAnthony — APPROVED** [review]

**Review by Juanro-Minne — APPROVED** [review]

File: src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class AccordionAdministrativeDocumentsComponent {
   selectedFile !: File;
   roles: string[] = [];
   isLoadingUpload: boolean = false;
-  allowedTypes = ['application/pdf'];
+  allowedTypes = ['application/pdf', 'image/png', 'image/jpeg'];
```
Comment (MatthewSchoeman) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts:38]: Should we not also include image/jpg if that is a thing. JPEG and JPG are two different file types even though they are files used for images

**Review by DewanAlberts — APPROVED** [review]

**Review by MatthewSchoeman — APPROVED** [review]

File: src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts (hunk 1 above)
Comment (ReinoRabbit) [reply src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts]: I removed the jpeg as the design only specifies jpg not jpeg. Small oversight

File: src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export class AccordionAdministrativeDocumentsComponent {
     if (this.allowedTypes.includes(this.selectedFile.type)) {
```
Comment (LlewellynAnthony) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts:101]: You're going to have to add another conditional check to determine if the uploadButtonIndex is the index of Written Approvals and if the allowed types are included. Else if only PDF is included. Else throw the snackbar message.

**Review by dvanhout89 — APPROVED** [review]

**Review by dvanhout89 — DISMISSED** [review]

File: src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts (hunk 2 above)
Comment (ReinoRabbit) [reply src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts]: Made use of a conditional check

**Review by LlewellynAnthony — APPROVED** [review]

**Review by MathildaShongwe15 — APPROVED** [review]

**Review by DewanAlberts — APPROVED** [review]

**Pull request merged**